rs/calculator/battle_state.py

```diff
@@ -892,6 +911,7 @@ def add_player_block(self, amount: int):
         if amount:
             self.player.block += amount
             self.trigger_block_effects()
+            self.player.block = min(self.player.block, 990)  # lower than 999 to avoid flailing against Beat of Death
```

Comment:
for efficiencies sake we could put this at like 500?

Reply:
Think I'd like to err on the side of it being as close to technically correct as possible and rely on other systems for not wasting our time too much instead of building in (more) intentional errors for efficiency